include/led-matrix-c.h

```diff
@@ -87,6 +87,12 @@ struct RGBLedMatrixOptions {
    * Corresponding flag: --led-pwm-bits
    */
   int pwm_bits;
+  
+  /* Set S-PWM segment bits used for output. Default is 8. Higher require less CPU and
+   * decreases refresh-rate.
+   * Corresponding flag: --led-seg-bits
+   */
+  int seg_bits;
```

Comment:
Even though these values are related to pwm_bits, we should put them to the end of the struct (also on the C side).

That way we preserve binary compatibility in case someone has compiled against an old version of the library but links against a newer version (well, it is not a guarantee that it will work, but at least we should make an effort).